libs/stdlib/iec61131-st/date_time_numeric_functions.st

```diff
@@ -1,424 +1,402 @@
-(********************
+(* *******************
```

Comment:
These new spaces in comment blocks does not seem right thing to do.

In https://github.com/PLC-lang/rusty/issues/837 is documentation format if I''m not mistaken.

Reply:
This is due to my auto-format on save settings in my editor and will be fixed before the PR goes out of draft